### DISCUSSION TITLE
can not open Settings on MacOS 12 Monterey

### DISCUSSION THREAD

**Discussion by falavu** [iterate-ch/cyberduck#15822] [Q&A]
Hello Community, 
please sorry for my cruel English – its not my Language. 
It is not possible to configure anything because i **can not open the Settings** – so i can not connect to a Server. 
The Finder-Menue does not open on Click. 
I deleted anything in ~/Library except my Licensekey. Restarted Finder and the App using the Taskman (restarts itself). 
Whats wrong? What can i do? 
Thank You very much for Hints and Solutions. 
Mountain Duck 1.7.1, MacOS 12.7.4 Monterey.

**Comment by dkocher** [comment]
Version 1.7.1 was released on 14 Feb 2017 while macOS Monterey on October 25, 2021. Therefore you cannot expect compatibility. Please upgrade your copy of Mountain Duck.
